server/api/build.go

```diff
@@ -54,7 +54,7 @@ func GetBuilds(c *gin.Context) {
 }
 
 func GetBuild(c *gin.Context) {
-	store_ := store.FromContext(c)
+	_store := store.FromContext(c)
```

Comment:
We could even use `store` here and in a lot of other cases, as redefining the variable should not be a problem. Not sure what is best :thinking:

Reply:
well I'll change that anyway ... later :)